### PR TITLE
Tickets

### PR DESCRIPTION
**Pull request by bnelson7**

### CODE REVIEW

File: frontend/src/components/tickets/ticket_activity_index.jsx [hunk 1]
```diff
@@ -63,7 +63,14 @@ class TicketActivityIndex extends React.Component {
 
             
 
-        return <div>{feedList}</div>;
+        return (
+          <div>
+            {this.props.ticket.lastUpdateSeenBy[0].firstName}{" "}
+            {this.props.ticket.lastUpdateSeenBy[0].lastName} last viewed the ticket
```
Comment (haydenlinder) [inline frontend/src/components/tickets/ticket_activity_index.jsx:69]: need the first 3 viewers, followed by a + and the number of remaining viewers

Comment (bnelson7) [reply frontend/src/components/tickets/ticket_activity_index.jsx:69]: k ill fix

**Comment by bnelson7** [comment]
the only thing left that would need to be fixed is the comma on the last viewed name

File: frontend/src/components/tickets/ticket_activity_index.jsx [hunk 2]
```diff
@@ -26,19 +28,15 @@ class TicketActivityIndex extends React.Component {
             ticketId: this.props.ticketId
         }));
 
-        // do you want a created at here 
-        // dont understand update and viewer
-
-        let updatesArr = ticket.updatedBy.map((actor, i) => ({
-            firstName: actor.firstName,
-            lastName: actor.lastName,
-            userId: actor._id,
```
Comment (haydenlinder) [inline frontend/src/components/tickets/ticket_activity_index.jsx:35]: these changes should have been accepted

File: frontend/src/components/tickets/ticket_form.jsx [hunk 3]
```diff
@@ -340,7 +340,8 @@ class TicketForm extends React.Component {
 
                 {this.props.ticketId !== "new" ? (
                 <ul className="activity-container">
-                    <h1 className="title">Comments and activity</h1>
```
Comment (haydenlinder) [inline frontend/src/components/tickets/ticket_form.jsx:343]: this one too

**Pull request closed without merging**